### DISCUSSION TITLE
Only certain DEXs are shown for ASAs?

### DISCUSSION THREAD

**Discussion by marucat27** [asastats/channel#303] [General]
I noticed when selecting an ASA, some would show one dex, others would show multiple.  Is there a reason for only showing certain dexs for ASAs?  For example, if you select Opulous, only "Pact swap" is shown.  Select ASASTATs, no dex shown. Smile Coin shows Humble, Tinyman, Algofi, and Pact.  Yieldly shows no dex?  This is on the left side of the site.  ![image](https://user-images.githubusercontent.com/109246416/197314926-542a7bad-8619-4d4f-8087-2de5912d001a.jpeg)![image](https://user-images.githubusercontent.com/109246416/197314935-a090a748-4699-4d08-8540-600bc103bb43.jpeg)![image](https://user-images.githubusercontent.com/109246416/197314941-2f33312e-6198-4f89-85ba-b9174fe6f2c0.jpeg)

**Comment by dragmz** [comment]
I think it's the maximum value engine that gives you an optimal route for swapping your ASA to ALGO. If that's the case then this is expected and depends on ASA liquidity on the supported DEXes.

**Reply by kerrilija** [reply]
@marucat27 I had similar questions, answered here:

https://github.com/asastats/channel/discussions/110#discussioncomment-3026363

**Comment by ipaleka** [comment]
DragMZ and kerrilija have provided answers and reading material, I'll just add that we had to choose one value and it has been decided that value is the maximum value.
